### PR TITLE
Adds two new kitchen signs

### PR DESCRIPTION
**Pull request by KingOfThePing**
This PR adds two new kitchen signs (so Event Horizon isnt alone anymore). Sprites were done by Lancer. They are also animated. 

Thematically one refers to Silversun, the other one to Eridani I. This PR also updates one of the bar signs to align with recent Eridani lore changes. 

paradiseSands
cityAlive

### CODE REVIEW

**Comment by KingOfThePing** [comment]
!review

File: code/game/objects/structures/barsign.dm [hunk 1]
```diff
@@ -92,89 +92,99 @@
 
 /decl/sign/double/bar/whiskey_implant
```
Comment (SleepyGemmy) [inline code/game/objects/structures/barsign.dm:93]: please add a newline inbetween each decl, if possible.

File: code/game/objects/structures/barsign.dm [hunk 2]
```diff
@@ -92,89 +92,99 @@
...
 	desc_fluff = "People's Preferred is the name of the drinking hole at Pluto's biggest spaceport. Everyone stops there and everyone loves it there. It's what the people prefer, apparently."
 /decl/sign/double/kitchen/event_horizon // Start of the kitchen signs. Don't mix it up.
 	name = "Event Horizon"
 	icon_state = "Event Horizon"
 	desc = "The SCCV Horizon's kitchen franchise sign."
 	desc_fluff = "The SCCV Horizon's dining area was the testing ground for the SCC to experiment with food franchising. The goal was to provide better food perparing processes, food quality and, of course, to maybe capitalize on this. To remember where it all started, the name 'Event Horizon' was chosen."
+/decl/sign/double/kitchen/paradise_sands
+	name = "Paradise Sands"
+	icon_state = "Paradise Sands"
+	desc = "A take on the sign of one of Silversun's most popular cafe & bistro, owned by Idris Incorporated."
```
Comment (SleepyGemmy) [inline code/game/objects/structures/barsign.dm:181]:
```suggestion
	desc = "A take on the sign of one of Silversun's most popular cafe- and bistro owned by Idris Incorporated."
```

File: code/game/objects/structures/barsign.dm [hunk 3]
```diff
@@ -92,89 +92,99 @@
...
+	desc_fluff = "Paradise Sands is the name of one of the most popular Cafe & Bistro places on all of Silversun. Located in a seculuded cove, you can brunch, drink and swim at a beach with white sand and a deep, azure ocean. Simply a paradise."
```
Comment (SleepyGemmy) [inline code/game/objects/structures/barsign.dm:182]:
```suggestion
	desc_fluff = "Paradise Sands is the name of one of the most popular cafe- and bistro places on all of Silversun. Located in a secluded cove, you can brunch, drink, and swim at a beach with white sand and a deep, azure ocean; a paradise."
```

File: code/game/objects/structures/barsign.dm [hunk 4]
```diff
@@ -92,89 +92,99 @@
...
+/decl/sign/double/kitchen/city_alive
+	name = "City Alive"
+	icon_state = "City Alive"
+	desc = "City Alive is another popular restaurant chain, originating from Eridani I. It is famous for it's light shows."
```
Comment (SleepyGemmy) [inline code/game/objects/structures/barsign.dm:186]:
```suggestion
	desc = "City Alive is another popular restaurant chain, originating from Eridani I. It is famous for its light shows."
```

File: code/game/objects/structures/barsign.dm [hunk 5]
```diff
@@ -92,89 +92,99 @@
...
+	desc_fluff = "City Alive is a high class restaurant chain, dotted all over Eridani I and III. Especially on Eridani I they are also famous for their light shows in the evenings. These lights look like pulsating veins, making the city seem alive, when observed from orbit."
```
Comment (SleepyGemmy) [inline code/game/objects/structures/barsign.dm:187]:
```suggestion
	desc_fluff = "City Alive is a high class restaurant chain, dotted all over Eridani I and III. Especially on Eridani I they are also famous for their light shows in the evenings. These lights look like pulsating veins, making the city seem alive, especially when observed from orbit."
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by Alberyk — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**